GDrive/app/src/main/java/org/jboss/aerogear/android/cookbook/gdrive/MainActivity.java

```diff
@@ -119,11 +118,14 @@ private void retrieveFiles() {
                 .responseParser(gsonResponseParser)
                 .forClass(Files.class);
 
+        @SuppressWarnings("unchecked")
         Pipe<Files> documentsPipe = PipeManager.getPipe("files", this);
         documentsPipe.read(new Callback<List<Files>>() {
             @Override
             public void onSuccess(final List<Files> fileses) {
-                Toast.makeText(getApplicationContext(), getString(R.string.fetched, fileses.size()), Toast.LENGTH_LONG).show();
+                Toast.makeText(getApplicationContext(),
+                        getResources().getQuantityString(R.plurals.fetched, fileses.size()),
```

Comment:
Should be 
`getResources().getQuantityString(R.plurals.fetched, fileses.size()
, fileses.size())` to replace the %d with the actual number of files